lms/templates/ccx/enrollment.html

```diff
@@ -5,14 +5,14 @@
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
   <h2> ${_("Batch Enrollment")} </h2>
   <p>
-    <label for="student-ids">
+    <label id="label_student_ids" aria-hidden="true">
```

Comment:
I found issues using for attribute here , When I use for then user can read label 2 times , one as label and second as hint of text area